main.py

```
@@ -1,11 +1,273 @@
+        title = QLabel("Your Statistics")
+        title.setAlignment(Qt.AlignCenter)
+        title.setStyleSheet("font-size: 12px; font-weight: bold; padding: 2px;")
+        final_layout.addWidget(title)
+        final_layout.addWidget(stats)
+
+        return final_widget
+
+    def update_status(self, message: str):
+        self.status.showMessage(message, 4000)
+
+def count_files_and_storage(dir: str) -> Tuple[int, float]:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)